### ISSUE TITLE
ValueError when computing Jacobian in eager mode

### ISSUE THREAD

**Issue by TakeByEarn** [tensorflow/tensorflow#34260]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>
I'm trying to add jacobian regularization to our loss function for reduce the intputs of sensitivity.
but in tensorflow(version 1.14.0), it's hard to build the dynamic graph of jacobian regularization.
I try to use "GradientTape()" for get the jacobian matrix. but there is problem to get the weight parameters's gradient.  "raise ValueError("Tape is already recording.")", which is means we cannot use GradientTape() to get the jacobian regularization function. I guess there is another way to get it done. That's is build the dynamic graph of jacobian regularization function expliciltly. but this could be very hard for me. Is there any way to do this convenient？

**System information**
- TensorFlow version (you are using):
tensorflow version 1.14.0
- Are you willing to contribute it (Yes/No):
No, Cause i'm not very famililar with tensorflow's framework.

**Describe the feature and the current behavior/state.**
I want to use the jacobian regurlarization to my model in tensorflow. 
**Will this change the current api? How?**
yes,
**Who will benefit with this feature?**
expert, researcher.
**Any Other info.**

**Comment by ravikyram** [comment]
Do you have any use case that requires the feature you are interested in? Please feel free to submit a PR if you have use cases that supports that feature.Thanks!

**Comment by ravikyram** [comment]
@TakeByEarn 

Will it be possible to share minimal stand alone code to reproduce the issue reported here. Thanks!

**Comment by ravikyram** [comment]
@TakeByEarn 

Any update on this issue please. Thanks!

**Comment by jvishnuvardhan** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.
